client/verta/tests/test_cli/test_endpoint.py

```diff
@@ -410,6 +411,56 @@ def test_update_autoscaling(self, client, created_endpoints, experiment_run, mod
                 assert metric["parameters"][0]["value"] == "0.7"
 
 
+class TestDownload:
+    def test_download_context(self, experiment_run, model_for_deployment, registered_model, in_tempdir, created_registered_models, model_version):
+        np = pytest.importorskip("numpy")
+        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
+        experiment_run.log_requirements(['scikit-learn'])
+
+        artifact = np.random.random((36, 12))
+        experiment_run.log_artifact("some-artifact", artifact)
+
+        download_to_path = "context_cli.tgz"
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ['deployment', 'download', 'dockercontext', '--run-id', experiment_run.id, '--output',
+             download_to_path],
+        )
+        assert not result.exception
+
+        # can be loaded as tgz
+        with tarfile.open(download_to_path, 'r:gz') as f:
+            filepaths = set(f.getnames())
+
+        assert "Dockerfile" in filepaths
+
+        np = pytest.importorskip("numpy")
+        from sklearn.linear_model import LogisticRegression
+
+        classifier = LogisticRegression()
+        classifier.fit(np.random.random((36, 12)), np.random.random(36).round())
+        model_version.log_model(classifier)
```

Comment:
We can use the same model object as before, provided by the fixture
```suggestion
        model_version.log_model(model_for_deployment['model'], custom_modules=[])
```